core/app/services/spree/export/google_rss.rb

```diff
@@ -0,0 +1,144 @@
+require 'nokogiri'
+
+module Spree
+  module Export
+    class GoogleRss
+      class MissingStore < ActiveRecord::RecordNotFound
+        def initialize(msg="Store with id #{options.spree_store_id} does not exist.")
+          super(msg)
+        end
+      end
```

Comment:
Instead of creating new class in service object, I would like to suggest to use `prepend Spree::ServiceModule::Base` defined module and return fails by `failure` or `success` if everything works fine

here you can find out example of usage `ServiceModule` : https://github.com/spree/spree/blob/main/core/app/services/spree/cart/add_item.rb#L4 

here you can see how errors are returned: https://github.com/spree/spree/blob/main/core/app/services/spree/cart/add_item.rb#L38